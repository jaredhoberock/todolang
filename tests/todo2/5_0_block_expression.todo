var x: String = {
  print "statement 1";
  var x: String = "inside block";
  print x;
  var y: String = "OK";
  y
};

print "outside block";
print x;
