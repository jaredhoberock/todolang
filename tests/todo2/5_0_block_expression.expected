statement 1
inside block
outside block
OK
